packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/XDD/Document.java

```diff
@@ -0,0 +1,60 @@
+package software.uncharted.terarium.hmiserver.models.xdd;
+
+import java.util.HashMap;
+import java.util.Hashtable;
+import java.util.List;
+
+import javax.json.bind.annotation.JsonbProperty;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+/**
+ * XDD Document representation
+ */
+public class Document {
+
+	@JsonbProperty("_gddid")
+	public String _gddid;
```

Comment:
variables should follow the regular expression '^[a-z][a-zA-Z0-9]*$'. please change to `gddId`

Reply:
Sure. My initial version just focused on matching what is coming back from XDD

Reply:
done